src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -379,7 +597,6 @@ function isCtrlKey(e) {
 }
 pre.sf-dump .sf-dump-ellipsis {
     display: inline-block;
-    overflow: visible;
```

Comment:
Not related, but overridden some lines below. (https://github.com/symfony/symfony/blob/master/src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php#L386).
Is it worth it to be applied to lower branches?